src/components/VideoPlayer2.tsx

```diff
@@ -92,6 +92,52 @@ export const VideoPlayer: FunctionComponent<VideoPlayerProps> = ({
     return pipButtonContainer;
   };
 
+  // Fetch previously saved captions setting from localStorage
```

Comment:
Can you remove this comment?

Reply:
i added this comment to help other people understand these line of codes. anyways i'll remove it